app/src/main/res/values/strings.xml

```diff
@@ -480,4 +481,5 @@
   <string name="developer_options_force_crash">Force crash</string>
   <string name="all">All</string>
   <string name="mark_completed_uppercase">MARK COMPLETED</string>
+  <string name="network_selected">Network is Selected</string>
```

Comment:
Nit
`   <string name="network_selected">Network is selected</string>`

Reply:
Done.

Reply:
Resolving comment as the nit is fixed.